docs/docs/contributing/scala2-vs-scala3.md

```diff
@@ -0,0 +1,38 @@
+# Divergences between Scala 2 and Dotty
```

Comment:
This page, https://github.com/lampepfl/dotty/wiki/Known-incompatibilities-with-scalac and https://github.com/scala/scala-dev/issues/374 contain overlapping information that should be centralized.

Reply:
I added this task earlier here: lampepfl/dotty-knowledge#4